packages/workflows-shared/src/binding.ts

```diff
@@ -92,16 +92,30 @@ export class WorkflowHandle extends RpcTarget implements WorkflowInstance {
 		throw new Error("Not implemented yet");
 	}
 
-	public async status(): Promise<InstanceStatus> {
+	public async status(): Promise<
+		InstanceStatus & { __LOCAL_DEV_STEP_OUTPUTS: unknown[] }
+	> {
 		const status = await this.stub.getStatus(0, this.id);
-		const { logs } = await this.stub.readLogs();
-		// @ts-expect-error TODO: Fix this
+		const { logs } = (await this.stub.readLogs()) as { logs: unknown[] };
```

Comment:
Why do we type a log as `unknown` here? We could instead create a type assertion function.

```ts
type Log = {
  event: InstanceEvent,
  metadata: {
    result: unknown
  }
};

function assertHasLogs(obj: unknown): obj is { logs: Log[] } {
  // check for props
}
```

Could probably have just one TODO to remove once correct typings are added to `readLogs`.

Reply:
I've fixed the `readLogs` return type - there's just one @ts-expect-error on the binding return because, agh, workers-types are a bit wrong. I'll follow-up with another PR once binding types are fixed.